components/dash-core-components/README.md

```diff
@@ -18,7 +18,7 @@ $ python -m venv venv && . venv/bin/activate
 $ pip install -e .[dev,testing]  # in some shells you need \ to escape []
 
 # run the build process - this will build all of dash, including dcc
-$ npm i && npm run build
+$ npm ci && npm run build
```

Comment:
I'm not sure the component readmes (and CONTRIBUTING.md files) are still relevant - may have some useful info in them, but a lot of it is probably obsolete.

Reply:
It is mentioned [here](https://github.com/plotly/dash/blob/dev/CONTRIBUTING.md#development-of-dash-core-components-dash-html-components-and-dash_table) that contributors should look into these.
But yeah it would be nice to merge/update those.